load-test-framework/src/main/java/com/google/pubsub/clients/gcloud/CPSSubscriberTask.java

```diff
@@ -39,10 +39,13 @@
 
   private CPSSubscriberTask(StartRequest request) {
     super(request, "gcloud", MetricsHandler.MetricName.END_TO_END_LATENCY);
-    this.subscription = SubscriptionName.create(
-        request.getProject(), request.getPubsubOptions().getSubscription());
+    this.subscription =
+        SubscriptionName.create(request.getProject(), request.getPubsubOptions().getSubscription());
     try {
-      this.subscriber = Subscriber.defaultBuilder(this.subscription, this).build();
+      this.subscriber =
+          Subscriber.defaultBuilder(this.subscription, this)
+              .setParallelPullCount(Runtime.getRuntime().availableProcessors() * 5)
```

Comment:
Why is this called parallel pull count when it is doing StreamingPull?

Reply:
I'm not quite sure what you're asking. We are using StreamingPull and the number determines how many connections we create to pull in parallel.